functions/gameDesigner.js

```diff
@@ -0,0 +1,32 @@
+export default function gameDesigner() {
+  let thing = [];
+  for (let i = 0; i < 150; i++) {
+    if (i === 0 || i === 1 || i === 2) {
```

Comment:
You could also write:

```suggestion
    if (i >= 0 || i <= 2) {
```